packages/paste-core/components/timeline/src/index.tsx

```diff
@@ -0,0 +1,5 @@
+export { Timeline } from "./Timeline";
+export { TimelineItem } from "./TimelineItem";
+export { TimelineContext } from "./TimelineContext";
+export { TimelineItemIcon } from "./TimelineItemIcon";
+export type { TimelineProps, TimelineItemProps, TimelineItemIconProps, Orientation } from "./types";
```

Comment:
nit: I don't think Orientation will need to be exported to users as it will still use the type values for props
```suggestion
export type { TimelineProps, TimelineItemProps, TimelineItemIconProps } from "./types";
```